recipes/python-measurement/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "measurement" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  # building using the release on github has an issue. see: https://github.com/coddingtonbear/python-measurement/issues/27
+  # from pypi
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 36ca385ffdccf140a75a7e1d816a4df97a6dd255f16fd2f53dd7ab43632a8835
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - pbr
+  run:
+    - python
+    - six >=1.0
+    - sympy >=0.7.3
+
+test:
+  imports:
+    - measurement
+  source_files:
```

Comment:
You need to add a `commands` section to run these tests else the source files aren’t needed

Reply:
Good to know. I was playing around with this guide on [testing](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=test%20files#test-section) files.